server/lib/insightProviders/pageInsightProviders/webPageTest.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const debug = require('debug')('perf-widget:lib:insightsProviders:pageInsightProviders:webPageTest'); // eslint-disable-line no-unused-vars
+const bluebird = require("bluebird");
+const WebPageTest = require('webpagetest');
+const denodeify = require('denodeify');
+const wpt = new WebPageTest(process.env.WEB_PAGE_TEST_SERVER, process.env.WEB_PAGE_TEST_API_KEY);
```

Comment:
We need to either use the internal WebPageTest server or request an API key from webpagetest.org and ask if the per day limit can be raised higher than the default 200 requests a day.